mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_management_server.go

```diff
@@ -0,0 +1,31 @@
+package backupdr
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func dataSourceGoogleCloudBackupDRService() *schema.Resource {
+
+	dsSchema := tpgresource.DatasourceSchemaFromResourceSchema(ResourceBackupDRManagementServer().Schema)
+	tpgresource.AddRequiredFieldsToSchema(dsSchema, "name", "location", "network")
```

Comment:
I wonder if `network` is needed for getting the resource.
Plus, I see that there's only `location` specified in test 
```
data "ggoogle_backup_dr_management_server" "foo" {
  location = google_backup_dr_management_server.foo.location
}
```
I believe all required fields are needed to read the resource

Reply:
Management server is a regional resource. There is only one resource per region. Hence added only location. Removed other fields from AddRequiredFieldsToSchema